tests/modules/sightings/resources/test_custom_fields.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+
+from tests.modules.sightings.resources import utils as sighting_utils
+from tests.extensions.edm import utils as edm_utils
+from tests import utils as test_utils
+
+
+CONFIG_PATH = '/api/v1/configuration/default'
+
+
+def test_custom_fields_on_sighting(
+    db, flask_app_client, researcher_1, test_root, staff_user, admin_user
+):
+    from app.modules.sightings.models import Sighting
+    from app.modules.encounters.models import Encounter
+    from app.modules.assets.models import Asset
+    from app.modules.asset_groups.models import AssetGroup
+    import datetime
+
+    cfd_id = edm_utils.custom_field_create(flask_app_client, admin_user, 'test_cfd')
+    assert cfd_id is not None
+
+    # we should end up with these same counts (which _should be_ all zeros!)
+    orig_ct = test_utils.multi_count(db, (Sighting, Encounter, Asset, AssetGroup))
```

Comment:
Every usage of this multi_count has the same params and each test that does so has to import all of the models. 
Would a test_utils.all_count(db) be better and then only that function would need to do all of the immports?

Reply:
at this point, yeah.  it seems like (speaking for myself anyway!) i just test all of these classes each time.   might chunk this into the next pr cuz this one is getting kinda stale, heh